spec/shopify_transporter/pipeline/magento/order/line_items_spec.rb

```diff
@@ -26,19 +26,22 @@ module ShopifyTransporter::Pipeline::Magento::Order
       it 'extracts line item attributes from an input hash' do
         magento_order_line_items = 2.times.map do |i|
           FactoryBot.build(:magento_order_line_item,
-            qty_ordered: "qty ordered #{i}",
+            qty_ordered: "5",
+            qty_shipped: "5",
             sku: "sku #{i}",
             name: "name #{i}",
             price: "price #{i}",
             tax_amount: "10",
             tax_percent: "12",
+            discount_amount: "20",
+            is_virtual: "0"
```

Comment:
nit: i believe you haven't run rubocop on this, since it would swap it to single-quotes? unsure